qualtran/bloqs/util_bloqs.py

```diff
@@ -50,8 +50,31 @@
     from qualtran.simulation.classical_sim import ClassicalValT
 
 
+class UtilBloq(Bloq, metaclass=abc.ABCMeta):
+    """Base class for util bloqs.
+
+    1. Trivial controlled - pass through the control register.
+    2. Do not affect T complexity.
+    """
```

Comment:
grammar & style in this docstring.

Reply:
polished the docstring, how does it look now?